packages/core/core/src/ReporterRunner.js

```diff
@@ -46,6 +50,8 @@ export default class ReporterRunner {
 
     if (this.options.patchConsole) {
       patchConsole();
+    } else {
+      unpatchConsole();
```

Comment:
Why do we need to unpatch the console when patchConsole is false? How could it have been patched before this?

Reply:
In tests it could have been patched in a previous test...

Probably could happen in any framework built on top of parcel or anything